config/locales/en.yml

```diff
@@ -155,7 +155,7 @@ en:
     jobs:
       unable_to_publish: We were unable to publish this job. Make sure that you have filled all mandatory fields
     sign_in:
-      unauthorised: You are not authorised to act on behalf of this school
+      unauthorised: Your school has not been onboarded yet. Please try again when it has.
```

Comment:
@despo This doesn't tell the user _how_ to resolve their issue and become 'on-boarded' which risks a frustrating experience. I think we could make this better by giving an instruction to contact us, perhaps:

> Your school has not been invited yet, you can register your interest by emailing isobel.croot@digital.education.gov.uk. You will be able to try again after receiving your invitation.

Reply:
We were just talking about this @Ellayanor, do you have anything further to add?

Reply:
I think you're right @tahb - worth checking with Isobel whether the generic email contact is imminent, but otherwise I like what you've suggested as a starting point, we definitely need to give users some idea of how to proceed. Michael needs input into this before we deploy it.

Reply:
https://trello.com/c/BbGZrTGU

Reply:
I've pointed Michael at this thread.